.github/workflows/hlwm.yml

```diff
@@ -210,3 +210,38 @@ jobs:
           else
             echo "Skipping comment posting because of absence of secret"
           fi
+
+  build-doc:
+    name: Build Documentation and website
+    runs-on: ubuntu-latest
+    env:
+      BUILD: build-${{ github.run_number }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: install asciidoc
+        run: sudo apt-get install --no-install-recommends asciidoc xsltproc docbook-xsl
```

Comment:
Should we just add this to ci/Dockerfile.ci-focal?

Reply:
Yes, then we could have a 'full build' there.

Reply:
I mean: we could do this eventually. I still think it's ok to have a separate job for the time being.

Reply:
Sure!